systems/velero/values.tmpl.yaml

```diff
@@ -0,0 +1,39 @@
+velero:
+{{- if and (hasKey .Requirements "velero") (hasKey .Requirements.velero "namespace") }}
+  {{- if .Requirements.velero.namespace }}
+  enabled: true
+  {{- else }}
+  enabled: false
+  {{- end }}
+{{- else }}
+  enabled: false
+{{- end }}
+  rbac:
+    create: true
+  credentials:
+    useSecret: true
+    existingSecret: velero-secret
+{{- if eq .Requirements.cluster.provider "gke" }}
+  configuration:
+    provider: gcp
+    backupStorageLocation:
+      name: gcp
+      bucket: {{ .Requirements.storage.backup.url | removeScheme | quote }}
+{{- else if or (eq .Requirements.cluster.provider "aws") (eq .Requirements.cluster.provider "eks") }}
+  configuration:
+    provider: aws
+    backupStorageLocation:
+      name: aws
+      bucket: {{ .Requirements.storage.backup.url | removeScheme | quote }}
+      config:
+        region: {{ .Requirements.cluster.region | quote }}
+{{- else if eq .Requirements.cluster.provider "azure" }}
+  configuration:
+    provider: azure
+    backupStorageLocation:
+      name: azure
+      bucket: {{ .Requirements.storage.backup.url | removeScheme | quote }}
+      config:
+        storageAccount: {{ .Requirements.veloro.serviceAccount | quote }}
```

Comment:
there is a type here `veloro` instead of `velero`

Reply:
many thanks for the catch!